src/test/kotlin/com/aroundme/likes/LikeServiceDatabaseTest.kt

```
@@ -0,0 +1,54 @@
+package com.aroundme.likes
+
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest
+import org.springframework.test.context.ActiveProfiles
+
+@DataJpaTest
+@ActiveProfiles("test")
+class LikeServiceDatabaseTest {
+
+    @Autowired
+    private lateinit var likeRepository: LikeRepository
+
+    private lateinit var likeService: LikeService
+
+    @BeforeEach
+    fun setup() {
+        likeService = LikeService(likeRepository)
+    }
+
+    @Test
+    fun `should successfully add a like to database`() {
+        val contentId = 1L
+        val userId = 100L
+
+        likeService.addLike(contentId, userId)
+
+        val savedLike = likeRepository.findByContentIdAndUserId(contentId, userId)
+        assertNotNull(savedLike)
+        assertEquals(contentId, savedLike?.contentId)
+        assertEquals(userId, savedLike?.userId)
+    }
+
+    @Test
+    fun `should prevent duplicate likes`() {
```

How about rephrase like this? 

`should raise an exception when a pair of contentId and userId is already counted for likes`

---

Done